src/main/java/io/github/pylonmc/pylon/base/items/tools/Hammer.java

```diff
@@ -140,15 +148,19 @@ private static MiningLevel getMiningLevel(@NotNull NamespacedKey key) {
         ).get(key);
     }
 
+    private static @NotNull @Unmodifiable List<ItemStack> hammersWithMiningLevelAtLeast(@NotNull MiningLevel level) {
+        return HAMMER_STACKS.stream()
+                .filter(item -> ((Hammer) fromStack(item)).miningLevel.isAtLeast(level))
+                .toList();
```

Comment:
By that I meant iterating all hammers in the registry, cause an addon might add a custom hammer